docs/content/kv_commands.md

```diff
@@ -7,19 +7,75 @@ The `kv` subcommand allows you to store application data in the Cloudflare netwo
 * have a Wrangler project set up with your `account_id` configured in the `wrangler.toml`
 * call commands from within a Wrangler project directory.
 
+## `kv:namespace`
+
+### `create`
+
+Creates a new namespace.
+
+#### Usage
+
+```console
+$ wrangler kv:namespace create "new kv namespace"
+🌀  Creating namespace with title "worker-new kv namespace"
+✨  Success: WorkersKvNamespace {
+    id: "e29b263ab50e42ce9b637fa8370175e8",
+    title: "worker-new kv namespace",
+}
+✨  Add the following to your wrangler.toml:
+kv-namespaces = [
+         { binding: "new kv namespace", id: "e29b263ab50e42ce9b637fa8370175e8" }
+]
+```
+
+### `list`
+
+Outputs a list of all KV namespaces associated with your account id.
+
+#### Usage
+
+```console
+$ wrangler kv:namespace list
+🌀  Fetching namespaces...
+✨  Success:
++---------------+----------------------------------+
+| TITLE         | ID                               |
++---------------+----------------------------------+
+| New Namespace | f7b02e7fc70443149ac906dd81ec1791 |
++---------------+----------------------------------+
+```
+
 Most `kv` commands require you to specify a namespace. A namespace can be specified in two ways:
-1. With a `--binding`: 
+
+1. With a `--binding`:
```

Comment:
I still feel like it's worth having this binding and namespace-id information up front, given that it applies to nearly all kv subcommands. What if someone ctrl-fs to a section of the docs and has no idea what these two arguments are?